### PR TITLE
Add integration testing with CVXPY

### PR DESCRIPTION
**Pull request by mkoeppe**
This adds a GH Actions workflow testing with CVXPY main HEAD. Test run at https://github.com/mkoeppe/CyLP/runs/5618813514?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/ci-cvxpy.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: Integration testing with CVXPY
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    tags:
+      - '*'
```
Comment (tkralphs) [inline .github/workflows/ci-cvxpy.yml:8]: This has to be `'**'` to capture tag names containing a `/`. Or it could also just be `'releases/*'`.

Comment (tkralphs) [reply .github/workflows/ci-cvxpy.yml:8]: But why not run this on every commit? I guess if all changes come in through PR's, then this is fine, but I get lazy sometimes and commit directly to master :).

Comment (mkoeppe) [reply .github/workflows/ci-cvxpy.yml]: Thanks, done

**Comment by tkralphs** [comment]
Forgive me if I missed it, but I didn't actually see where CyLP itself is getting tested in all of this. It looks like most of this testing would not be relevant to CyLP. I guess I'm missing the point here.

**Comment by mkoeppe** [comment]
CVXPY uses CyLP to interface CBC. Search for "CBC" in the output from the pytest run and

**Comment by tkralphs** [comment]
Great, thanks!

**Pull request merged**